lib/consts.ts

```diff
@@ -13,3 +13,7 @@ export const SUGGESTIONS = [
   "What did I create this week???",
   "What's my Zora score???",
 ]
+
+import manifest from '@/public/manifest.json';
+
+export const APP_DESCRIPTION = manifest.description;
```

Comment:
Why repeat the export in `consts`?
- why not directly import `manifest` in `layout`?

principle
- less is more
- simple is best
- small is beautiful